tests/ssl-opt.sh

```diff
@@ -1401,6 +1401,22 @@ run_test    "Renegotiation: client-initiated" \
             -s "=> renegotiate" \
             -S "write hello request"
 
+# Checks if the server parses the SigAlgs Extension. Valid only if an MD
+# algorithm stronger than SHA1 is enabled in config.h
+run_test    "Renegotiation: Signature Algorithms regression, client" \
+            "$P_SRV debug_level=3 exchanges=2 renegotiation=1 auth_mode=optional" \
+            "$P_CLI debug_level=3 exchanges=2 renegotiation=1 renegotiate=1" \
+            0 \
+            -c "client hello, adding renegotiation extension" \
+            -s "received TLS_EMPTY_RENEGOTIATION_INFO" \
+            -s "found renegotiation extension" \
+            -s "server hello, secure renegotiation extension" \
+            -c "found renegotiation extension" \
+            -c "=> renegotiate" \
+            -s "=> renegotiate" \
+            -S "client hello v3, signature_algorithm ext: 2" \
```

Comment:
How obvious is it that "signature_algorithm ext: 2" is SHA-1? Does this warrant a comment?